core/src/main/java/org/frankframework/jdbc/XmlQuerySender.java

```diff
@@ -86,6 +86,7 @@ public class XmlQuerySender extends DirectQuerySender {
 	public static final String TYPE_DATETIME_PATTERN = "yyyy-MM-dd HH:mm:ss.SSS";
 	public static final String TYPE_XMLDATETIME = "xmldatetime";
 
+	@Getter
```

Comment:
`CONTRIBUTING.MD` says that this shouldn't be put on the class, but on the specific attributes

Reply:
Yeah, that's true. But I see now, that this class is `public` but not used outside, so I'll make it `private`. That will make the arguments to not do so go away.